core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/GoogleGroupSyncMonitor.scala

```diff
@@ -79,15 +81,16 @@ class GoogleGroupSyncMonitorActor(val pollInterval: FiniteDuration, pollInterval
 
   self ! StartMonitorPass
 
-  // fail safe in case this actor is idle too long
-  setReceiveTimeout((pollInterval + pollIntervalJitter) * 10)
+  // fail safe in case this actor is idle too long but not too fast (1 second lower limit)
+  setReceiveTimeout(Seq((pollInterval + pollIntervalJitter) * 10, 1 second).max)
```

Comment:
this is not right, I want the old value the way it was with a minimum of 1s to avoid quantum effects. The jitter value is not actually doing any jitter in this case, this is just a fail safe to make sure actors don't get stuck.